dask/dataframe/io/sql.py

```diff
@@ -1,8 +1,9 @@
 import numpy as np
 import pandas as pd
 
-from ... import delayed
+import dask
 from .io import from_delayed, from_pandas
+from ... import delayed
```

Comment:
Note that in the Dask codebase itself we have tended towards using relative imports.  

You're right that in this particular case it might be ok (the delayed module is probably easier to strictly import than this module), but all of our current imports are relative, so it might be good to maintain consistency.